ui/src/app/views/workflow/run/node/workflow.run.node.component.ts

```diff
@@ -90,6 +90,15 @@ export class WorkflowNodeRunComponent {
                 this.subNodeRun = this._workflowEventStore.selectedNodeRun().subscribe(wnr => {
                     this.nodeRun = wnr;
                     if (this.nodeRun) {
+                        if (PipelineStatus.isDone(wnr.status)) {
+                            // Usefull to load tests and artifacts
+                            this._workflowRunService.getWorkflowNodeRun(
+                                this.project.key,
+                                this.workflowName,
+                                this.nodeRun.num,
+                                this.nodeRun.id
+                            ).subscribe((wfNodeRun) => this.nodeRun = wfNodeRun);
```

Comment:
You must not set nodeRun like this but via workflowEventStore.
Push the result by with _workflowEventStore.setSelectedNodeRun(this.nodeRun, false) and do something to avoid infiniteLoop.
By pushing into the store, the sidebar will be updated too.